tests/integration/features/istio/production/istio_installation.feature

```diff
@@ -67,3 +67,13 @@ Feature: Installing and uninstalling Istio module
     Then "Istio CR" is not present on cluster
     And Istio CRDs "should not" be present on cluster
     And Namespace "istio-system" is "not present"
+
+Scenario: Uninstallation of Istio module if Istio was manually deleted
```

Comment:
I again executed the scenario multiple times locally and I realized that it is flaky. 

The source of the flakiness is that sometimes the deletion of istio-system fails:
```
Error occurred during reconciliation of Istio installation	{"error": "namespaces \"istio-system\" not found"}
```

I think the steps in the current scenario are not correct. After uninstalling Istio manually we need to verify that istio-system and Istio CRDs are not present on the cluster any longer. Only then we can continue with triggering the uninstallation by the reconciler:

```
Scenario: Uninstallation of Istio module if Istio was manually deleted
    Given Istio CR "istio-sample" is applied in namespace "default"
    And Istio CR "istio-sample" in namespace "default" has status "Ready"
    And Namespace "istio-system" is "present"
    And Istio is manually uninstalled
    And Namespace "istio-system" is "not present"
    And Istio CRDs "should not" be present on cluster
    When "Istio CR" "istio-sample" in namespace "default" is deleted
    Then "Istio CR" is not present on cluster
```

Having the steps like that the scenario will constantly fail. We need to introduce the proposal from [my comment](https://github.com/kyma-project/istio/pull/178#pullrequestreview-1482743251) to fix it.